Dev 4

kateraawi opened the pull request:
modified:   .gitignore
	new file:   api/composer.json
	new file:   api/composer.lock
	modified:   api/dbconfig.php
	modified:   api/index.php
	modified:   api/models/Task.php
	modified:   api/models/User.php
	modified:   app/store/TaskStore.js

kateraawi committed:
Changes to be committed:
	modified:   .gitignore
	new file:   api/composer.json
	new file:   api/composer.lock
	modified:   api/dbconfig.php
	modified:   api/index.php
	modified:   api/models/Task.php
	modified:   api/models/User.php
	modified:   app/store/TaskStore.js

kateraawi committed:
Changes to be committed:
	modified:   api/index.php
	modified:   api/models/Task.php
	modified:   api/models/User.php
	modified:   app/view/front/MainController.js

ilyas-mspv commented on api/models/User.php:
```diff
@@ -3,116 +3,129 @@
     {
-        global $connection;
-        try {
+        global $em;
```
Логика не должна находится в сущности, тебе стоит сделать отдельный класс сервиса, в котором ты будешь получать данные, а в сущности должны прописываться только элементы, с которыми ты работаешь

ilyas-mspv committed:
Create project-template.md

kateraawi committed:
Interact with Task only via Controller. Fields users/tasks now public. Added TaskDto

kateraawi committed:
Merge branch 'dev' of https://github.com/kateraawi/test-exer into dev

kateraawi committed:
Changes to be committed:
	modified:   api/Controllers.php
	modified:   api/Services.php
	modified:   api/dbconfig.php
	modified:   api/dto/TaskDto.php
	new file:   api/dto/UserDto.php
	modified:   api/models/Task.php
	modified:   api/models/User.php
	new file:   api/proxies/__CG__TestExerModelUser.php
	new file:   api/proxies/__CG__User.php
	new file:   api/repository/TaskRepository.php
	modified:   app/store/TaskStore.js
	modified:   app/store/UserStore.js
	modified:   app/view/front/Main.js
	modified:   app/view/front/MainController.js
	modified:   bootstrap.json

kateraawi committed:
Commit

kateraawi committed:
Twig added

kateraawi committed:
Разнесены контроллеры и сервисы, добавлен combobox на фронтенде
